### DISCUSSION TITLE
Where can I get the commit for CVE-2023-5678 in 1.1.1

### DISCUSSION THREAD

**Discussion by nickychen123456** [openssl/openssl#23208] [Q&A] [answered]
We want to upgrade the openssl to 1.1.1w to fix some CVE issues, since the 1.1.1x has not been relased, we want to patch the commit 
https://git.openssl.org/gitweb/?p=openssl.git;a=commitdiff;h=710fee740904b6290fef0dd5536fbcedbc38ff0c but it cannot be open, just want to know why?

![image](https://github.com/openssl/openssl/assets/98001929/a602b463-a3b6-4cbd-af70-37649d77cfb2)

**Comment by mattcaswell** [comment, marked as answer]
Ah, that link should contain a link to our premium support page (like the link for 1.0.2 above) - we should fix that.

1.1.1 is now out of public support. See:

https://www.openssl.org/blog/blog/2023/09/11/eol-111/

The 1.1.1x release and its associated fixes, is only available to premium support customers.
